pubspec.yaml

```diff
@@ -9,24 +9,24 @@ topics:
   - rpc
 
 environment:
-  sdk: ^3.5.0
+  sdk: ^3.6.0
```

Comment:
We could go to `3.8.0` as the current stable is beyond that.. And then run a `dart pub upgrade --tighten`, as the lowest bounds don't build for me locally.

Reply:
sgtm; will do